src/overlays/actors/ovl_En_M_Fire1/z_en_m_fire1.h

```diff
@@ -7,7 +7,8 @@ struct EnMFire1;
 
 typedef struct EnMFire1 {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x50];
+    /* 0x144 */ ColliderCylinder collider;
+    /* 0x190 */ f32 unk_190;
```

Comment:
Small thing, but in MM, struct members usually don't have the underscore in their names